src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java

```diff
@@ -64,4 +67,20 @@ private void transformSwordBlockingPosition(AbstractClientPlayerEntity player, f
         matrices.multiply(RotationAxis.POSITIVE_Z.rotationDegrees(direction * 78.05F));
     }
 
+
+    @Inject(method = "renderFirstPersonItem",
+            at = @At(value = "INVOKE", target = "Lnet/minecraft/client/render/item/HeldItemRenderer;renderItem(Lnet/minecraft/entity/LivingEntity;Lnet/minecraft/item/ItemStack;Lnet/minecraft/client/render/model/json/ModelTransformationMode;ZLnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;I)V", ordinal = 1))
+    private void transformItemPosition1_7(AbstractClientPlayerEntity player, float tickDelta, float pitch, Hand hand, float swingProgress, ItemStack stack, float equipProgress, MatrixStack matrices, VertexConsumerProvider vertexConsumers, int light, CallbackInfo ci) {
+        // Modifies the handheld position to be slightly tilted like in 1.7 and prior
+        if (ProtocolTranslator.getTargetVersion().olderThanOrEqualTo(ProtocolVersion.v1_7_6) && !(stack.getItem() instanceof BlockItem)) {
```

Comment:
why the `stack.getItem() instanceof BlockItem`? Is this something the 1.7 client does in it's rendering code?

Reply:
No, but if I remove it, it breaks blocks position in the hand, this only affects items

Reply:
Then I would suppose the injection is at the wrong place?

Reply:
it isn't

Reply:
Looking at the ItemRenderer code in 1.7 they seem to special handle BlockItems.